x-pack/plugins/ml/public/application/routing/routes/datavisualizer/index_based.tsx

```diff
@@ -25,7 +25,7 @@ export const indexBasedRouteFactory = (
   navigateToPath: NavigateToPath,
   basePath: string
 ): MlRoute => ({
-  id: 'datavisualizer',
+  id: 'data_view_datavisualizer',
```

Comment:
Can we address the long-standing issue where the first page of the Data Visuaizer - Data View reuses the same route and breadcrumbs as selecting the data view in the job wizard:

![image](https://user-images.githubusercontent.com/7405507/149508936-67dc1984-ab25-4c97-a2eb-c908823d0798.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/123019/commits/409894898d46157ce3f48fdf0bc10f6018aebef5